content/faqs/I-found-a-bug!-What-do-I-do-now.md

```diff
@@ -1,8 +1,6 @@
 ---
-question: I found a bug! What do I do now?
-weight: 18
+question: I need help! I found a bug! What do I do now?
+weight: 26
 ---
 
-First, please make sure the bug hasn't already been reported by checking the current bug list [for Privacy Badger](https://github.com/EFForg/privacybadger/issues). If the bug hasn't yet been reported you can report the bug [here](https://github.com/EFForg/privacybadger/issues/new). If you don't have a GitHub account, we ask that you create one so that we can communicate with you about the bug and fix it more quickly.
-
-If you're not comfortable creating or don't want to create a GitHub account, you can also report the bug to [extension-devs@eff.org](mailto:extension-devs@eff.org).
+To get help or to report bugs, email [extension-devs@eff.org](mailto:extension-devs@eff.org). If you have a GitHub account, you can also report bugs on our [GitHub issue tracker](https://github.com/EFForg/privacybadger/issues).
```

Comment:
I like that we're more straightforward here, but maybe say something about how github is preferred if they are able? like "If you know the details of the bug you're reporting, the Github issue tracker is the best way to make sure it gets addressed quickly."

(If that is the case -- we might actually prefer the email, not sure)

Reply:
I'd rather hear from somebody over email and ask them to open an issue than have somebody struggle with GitHub just because it is "preferred".